spec/features/admin/index_spec.rb

```diff
@@ -33,12 +33,20 @@
 
       visit admin_index_path
 
-      expect(page).to_not have_content(invoice_2.id)
-
       expect(page).to have_content(invoice_1.id)
       expect(page).to have_content(invoice_3.id)
     end
 
+    it 'displays incomplete invoice information from oldest to newest' do
+      cust_1 = Customer.create!(first_name:"Hannah", last_name:"Warner")
+
+      invoice_1 = Invoice.create!(customer_id:"#{cust_1.id}", status:0)
+      invoice_2 = Invoice.create!(customer_id:"#{cust_1.id}", status:0)
+      invoice_3 = Invoice.create!(customer_id:"#{cust_1.id}", status:0, created_at:DateTime.yesterday)
+      visit admin_index_path
+      expect(page).to have_content(invoice_1.created_at.strftime("%A, %B %d, %Y"))
```

Comment:
Should invoice_2 and invoice_3 be expected here too? Also, I tested it out locally and this should work for at least one orderly test:

`expect(invoice_3.created_at.strftime("%A, %B %d, %Y")).to appear_before(invoice_1.created_at.strftime("%A, %B %d, %Y"))`